ledger/accountdb_test.go

```diff
@@ -712,8 +707,20 @@ func BenchmarkReadingAllBalances(b *testing.B) {
 		}
 	}
 	accountsInit(tx, updates, proto)
-	tx.Commit()
-	tx, err = dbs.rdb.Handle.Begin()
+	err = tx.Commit()
+	require.NoError(b, err)
+	return
+}
+
+func benchmarkReadingAllBalances(b *testing.B, inMemory bool) {
+	proto := config.Consensus[protocol.ConsensusCurrentVersion]
+	//b.N = 50000
+	dbs, _ := dbOpenTest(b, inMemory)
+	setDbLogging(b, dbs)
+	defer dbs.close()
+
+	benchmarkInitBalances(b, dbs, proto)
+	tx, err := dbs.rdb.Handle.Begin()
```

Comment:
the second part of this function doesn't seems to be needed for the benchmark, and looks like a test

Reply:
The part after b.ResetTimer() is the real benchmark, before that is just setup.
The hash might be meaningless. It was like that when I got here I just rearranged it.